src/main/java/net/pms/encoders/FFMpegVideo.java

```diff
@@ -356,13 +356,17 @@ public FFMpegVideo(PmsConfiguration configuration) {
 					transcodeOptions.add("-an");
 				} else if (type() == Format.AUDIO) {
 					// Skip
-				} else if (renderer.isTranscodeToAAC()) {
-					transcodeOptions.add("-c:a");
-					transcodeOptions.add("aac");
 				} else {
-					if (!customFFmpegOptions.contains("-c:a ")) {
-						transcodeOptions.add("-c:a");
-						transcodeOptions.add("ac3");
+					if (!customFFmpegOptions.contains("-(c:a|codc:a|acodec)")) {
```

Comment:
Should `codc:a` be `codec:a`?

Reply:
I don't think ```contains()``` support regex..

Reply:
The ```contains()``` definitelly not support regex. @onon765trb have you ever tested your PRs?